test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -11,26 +11,35 @@
 public class UtilsTest {
     @Test
     public void isAnyNull() {
+    	// list with only null objects
+    	assertTrue(Utils.isAnyNull((Object) null));
```

Comment:
Expectation was for your to write a test method such as isAnyNull(){...}

Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behaviour. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope